test/ui-test/lightspeedUiTestExplorer.ts

```diff
@@ -0,0 +1,271 @@
+import {
+  ActionsControl,
+  ActivityBar,
+  By,
+  ContextMenu,
+  EditorView,
+  InputBox,
+  ModalDialog,
+  SideBarView,
+  VSBrowser,
+  ViewControl,
+  ViewSection,
+  WebView,
+  WebviewView,
+  Workbench,
+} from "vscode-extension-tester";
+import {
+  getFixturePath,
+  expectNotification,
+  getModalDialogAndMessage,
+  getWebviewByLocator,
+  sleep,
+  updateSettings,
+  workbenchExecuteCommand,
+} from "./uiTestHelper";
+import { WizardGenerationActionType } from "../../src/definitions/lightspeed";
+import { PlaybookGenerationActionEvent } from "../../src/interfaces/lightspeed";
+import { expect } from "chai";
+import axios from "axios";
+
+export function lightspeedUIExplorerTest(): void {
+  describe("Test Lightspeed Explorer features", () => {
+    let workbench: Workbench;
+    let explorerView: WebviewView;
+    let modalDialog: ModalDialog;
+    let dialogMessage: string;
+    let sideBar: SideBarView;
+    let view: ViewControl;
+    let adtView: ViewSection;
+
+    before(async () => {
+      // Enable Lightspeed and open Ansible Light view on sidebar
+      workbench = new Workbench();
+      const settingsEditor = await workbench.openSettings();
+      await updateSettings(settingsEditor, "ansible.lightspeed.enabled", true);
+      await updateSettings(
+        settingsEditor,
+        "ansible.lightspeed.URL",
+        process.env.TEST_LIGHTSPEED_URL,
+      );
+      await updateSettings(
+        settingsEditor,
+        "ansible.lightspeed.suggestions.enabled",
+        true,
+      );
+      // Close settings and other open editors (if any)
+      await new EditorView().closeAllEditors();
+
+      // Set "UI Test" option for mock server
+      await axios.post(
+        `${process.env.TEST_LIGHTSPEED_URL}/__debug__/options`,
+        ["--ui-test"],
+        { headers: { "Content-Type": "application/json" } },
+      );
+    });
+
+    after(async function () {
```

Comment:
Please remove the teardown functions. Since we destroy the VSCode instance just after, we don't really care about that and it's a good way to reduce the risk of unexpected failures.